lib/dynamodb-integration.js

```diff
@@ -33,35 +35,49 @@ const popEventsToSend = (logEvents, cb) => {
     return logEvents.splice(0, entryIndex);
 }
 
-const buildPayload = (tableName, streamName, events) => {
+const buildPayload = (tableName, streamName, events, additionalAttributesSchema ) => {
+    var additionalAttributes = additionalAttributesSchema ? Object.entries(additionalAttributesSchema) : [];
     return {
-        RequestItems: {
-            [tableName]: events.map(event =>
-                ({
-                    PutRequest: {
-                        Item: {
-                            id: {
-                                "S": streamName
-                            },
-                            timestamp: {
-                                "N": event.timestamp.toString()
-                            },
-                            message: {
-                                "S": event.message
-                            },
-                        }
-                    }
-                })
-            )
-        }
+      RequestItems: {
+        [tableName]: events.map((event) => {
+          const rawMessage = event.rawMessage;
+          const extraValues = {};
+          if (additionalAttributesSchema) {
```

Comment:
I think this `if` is redundant. If `additionalAttributesSchema` is not truthy, then `additionalAttributes` is an empty array, and the `forEach` does nothing

Reply:
true https://github.com/env0/winston-aws-dynamodb/pull/18/commits/33fba3b5d6587b795a54411f10d16b3b4a47c493